src/test/components/Root-history.test.js

```diff
@@ -12,39 +12,27 @@ import mockCanvasContext from '../fixtures/mocks/canvas-context';
 import { getProfileUrlForHash } from '../../actions/receive-profile';
 import { blankStore } from '../fixtures/stores';
 import { getProfileFromTextSamples } from '../fixtures/profiles/processed-profile';
-import {
-  mockWindowLocation,
-  mockWindowHistory,
-} from '../fixtures/mocks/window-navigation';
+import { autoMockFullNavigation } from '../fixtures/mocks/window-navigation';
 import { coerceMatchingShape } from '../../utils/flow';
 import { makeProfileSerializable } from '../../profile-logic/process-profile';
 
 import type { SerializableProfile } from 'firefox-profiler/types';
 
 describe('Root with history', function() {
-  // Cleanup the tests through a side-effect.
-  let _cleanup;
-  afterEach(() => {
-    if (!_cleanup) {
-      throw new Error('Expected the setup function to create a cleanup step.');
-    }
-    _cleanup();
-  });
-
   type TestConfig = {|
     profileHash?: string,
   |};
 
+  autoMockFullNavigation();
```

Comment:
I find it easier to handle rather than handling a cleanup function in all test files that would need it.